ludwig/model_export/onnx_exporter.py

```diff
@@ -0,0 +1,39 @@
+import os
+from abc import ABC, abstractmethod
+
+import onnx
+import torch
+
+from ludwig.api import LudwigModel
+from ludwig.model_export.base_model_exporter import LudwigTorchWrapper
+
+
+class OnnxExporter(ABC):
+    def export_classifier(self, model_id, model_path, export_path, input_model_name, output_model_name):
```

Comment:
Some of these params are not being used. It think the function signature should look like:

```
def export(self, model_path, export_path)
```

I don't think it's necessary to include `_classifier` in the function name